homeassistant/components/spokestack_tts/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "spokestack_tts",
+  "name": "Spokestack Text-to-Speech",
+  "documentation": "https://www.home-assistant.io/integrations/spokestack_tts",
+  "requirements": ["numpy==1.19.2", "cython==0.29.21", "spokestack==0.0.16"],
```

Comment:
Why pin numpy and cython ? That's going to be a recipe for disaster with our 1000 other integrations.

Reply:
I originally pinned numpy and cython because of the [docs](https://developers.home-assistant.io/docs/creating_platform_code_review/#1-requirements). When I run `script/gen_requirements_all.py` I get an error also linking to this section of the docs.

Reply:
The `spokestack` Python package should define it's own requirements.

Reply:
That makes sense. Thank you for explaining. 😃